Content.Server/CartridgeLoader/Cartridges/LogProbeCartridgeComponent.cs

```diff
@@ -1,6 +1,6 @@
 using Content.Shared._CD.CartridgeLoader.Cartridges; // CD
 using Content.Shared.CartridgeLoader.Cartridges;
-using Content.Shared.Paper;
+﻿using Content.Shared.Paper;
```

Comment:
Upstream has an invisible [ZWNBSP](https://en.wikipedia.org/wiki/Word_joiner) at the start of this line :)